pallets/msa/src/lib.rs

```diff
@@ -1206,8 +1209,8 @@ impl<T: Config> Pallet<T> {
 		// Clear the previous bucket block set
 		let multi_removal_result = <PayloadSignatureRegistry<T>>::clear_prefix(
 			prior_bucket_num,
-			T::MaxSignaturesPerBucket::get(),
-			None,
+			T::MaxSignaturesPerBucket::get(), // I couldn't find any limitation for the number of PayloadSignatures that can be registered
+			None, // If there are more than 'MaxSignaturesPerBucket' items, you should check 'multi_removal_result.maybe_cursor' and handle the situation.
```

Comment:
@shannonwells Could you check on these potential overflows in the signature buckets?

Reply:
Tracking in #797